core/src/main/java/org/frankframework/filesystem/Samba2FileSystem.java

```diff
@@ -166,24 +166,22 @@ public void open() throws FileSystemException {
 
 	@Override
 	public void close() throws FileSystemException {
-		super.close();
-		try {
-			if(diskShare != null) {
+		if (diskShare != null) {
+			try {
 				diskShare.close();
+			} catch (IOException e) {
+				log.warn("error closing diskShare: {}", diskShare, e);
 			}
-			if(session != null) {
```

Comment:
Waarom is het sluiten van de session eigenlijk niet (meer) nodig?

Reply:
Is wel nodig, maar dat gebeurt gewoon al als de client sluit, op regel 177.